### ISSUE TITLE
Feature suggestion: goodness metric from model building 

### ISSUE THREAD

**Issue by flikka** [equinor/gordo-components#77]
Provide a metric (or several) from a model building run. Could be just the auto-encoder avg error or something, or a "overfitting indicator" where the difference between (ratio or similar) the results on training data and some test data is reported, stored, and provided as metadata. Could be a separate test period, or a cross validation / train test type split.

Need to discuss this with the whole team, and then implement. Should also be used when assessing new architectures or hyperparameters.
